src/protocollogin.cpp

```diff
@@ -66,9 +66,6 @@ void ProtocolLogin::getCharacterList(const std::string& accountName, const std::
 		output->addByte(0);
 	}
 
-	//Update premium days
-	Game::updatePremium(account);
```

Comment:
If you remove this, the first login during a day may show incorrect premium status because time is not recalculated before character list is sent.